presto-password-authenticators/src/main/java/io/prestosql/plugin/password/filebasedauthenticator/ForwardingPasswordAuthenticator.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.password.filebasedauthenticator;
+
+import io.prestosql.spi.security.PasswordAuthenticator;
+
+import java.security.Principal;
+import java.util.function.Supplier;
+
+import static java.util.Objects.requireNonNull;
+
+public abstract class ForwardingPasswordAuthenticator
```

Comment:
I don't think we need this class. The interface only has one method, so it's easy to implement using a lambda.

Reply:
Done

Reply:
Done